Test/git-issues/git-issue-425.dfy

```diff
@@ -0,0 +1,23 @@
+// RUN: %dafny /compile:0 "%s" > "%t"
+// RUN: %dafny /noVerify /compile:4 /compileTarget:cs "%s" >> "%t"
+// RUN: %dafny /noVerify /compile:4 /compileTarget:js "%s" >> "%t"
+// RUN: %dafny /noVerify /compile:4 /compileTarget:go "%s" >> "%t"
+// RUN: %dafny /noVerify /compile:4 /compileTarget:java "%s" >> "%t"
+// RUN: %diff "%s.expect" "%t"
+
+method M() returns (x: int, ghost y: int) {
+  return 42, 43;
+}
+
+datatype Color = Red | Blue
+  
+method Main() {
+  var x0, y0 := M();  // this is fine: x0 is compiled, y0 is ghost
+  print x0, "\n";
+  var c: Color;
```

Comment:
Variable `c` gets an arbitrary value. If the compiler changes in the future to pick this value a different way, then that would needlessly break this test. It's better to assign a particular value, since the test is not trying to test the arbitrary-ness of the choice.
```suggestion
  var c := Red;
```